src/components/pages/Home/index.js

```diff
@@ -76,42 +77,72 @@ const IndexLayout = ({
         <DesktopIntroContainer>
           <HomeIntroContainer>
             <HomeIntroContentContainer>
-              <Fade bottom distance="30px">
-                {homeIntroGraphic ? (
-                  <img
-                    src={homeIntroGraphic}
-                    alt="home intro graphic"
-                    className="Home__Intro-Graphic"
-                  />
-                ) : (
-                  ''
-                )}
-                {homeIntroTitle ? (
-                  <HomeIntroTitle as="h1" type="heading1">
-                    {homeIntroTitle}
-                  </HomeIntroTitle>
-                ) : (
-                  ' '
-                )}
-                {homeIntroText ? (
-                  <HomeIntroText as="p" type="smallText700">
-                    {homeIntroText}
-                  </HomeIntroText>
-                ) : (
-                  ''
-                )}
-                {homeIntroLink ? (
-                  <HomeIntroLink
-                    to={`#${ministriesUid}`}
-                    className="Home__Intro-link"
+              <HomeIntroBodyContainer>
+                <Fade bottom distance="30px">
+                  {homeIntroGraphic ? (
+                    <img
+                      src={homeIntroGraphic}
+                      alt="home intro graphic"
+                      className="Home__Intro-Graphic"
+                    />
+                  ) : (
+                    ''
+                  )}
+                  {homeIntroTitle ? (
+                    <HomeIntroTitle as="h1" type="heading1">
```

Comment:
very important not to have more then 1 h1 per <section> or page itself also please go through your title component and check that all title have a heading to them and they are not default <span>